### ISSUE TITLE
tensorflow-aarch64 with keras

### ISSUE THREAD

**Issue by SHFRGRP** [tensorflow/tensorflow#15416]
![screenshot_2017-12-17-15-05-30-239_ru iiec pydroid3](https://user-images.githubusercontent.com/31110466/34079284-0cd76cc4-e33c-11e7-8442-438e88e1a38d.png)
![screenshot_2017-12-17-15-04-20-194_ru iiec pydroid3](https://user-images.githubusercontent.com/31110466/34079285-0d1d5edc-e33c-11e7-9109-fbfdce775814.png)
![screenshot_2017-12-17-15-04-36-922_ru iiec pydroid3](https://user-images.githubusercontent.com/31110466/34079287-0d65de50-e33c-11e7-9b62-9d5b5027b131.png)

Hi! I want to use tensorflow library as backend of keras on my android device. I found and installed pydroid3, keras and tensorflow-aarch64. How can i use it? When i run code it throws exception "no module named tensorflow". Any ideas?

Ok, i will.
I used Keras. When i import, it throws exception.
My OS is Android 7.1.2 MIUI 9, CPU arch - aarch64 (arm), GPU is Snapdragon 505. Using PyDroid3 app
I installed tensorflow-aarch64 1.2 from pip.
No bazel.
No cuDNN.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by SHFRGRP** [comment]
Ok, i will.
I used Keras. When i import, it throws exception.
My OS is Android 7.1.2 MIUI 9, CPU arch - aarch64 (arm), GPU is Snapdragon 505.
I installed tensorflow-aarch64 1.2 from pip.
No bazel.
No cuDNN.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue? Please update the label and/or status accordingly.

**Comment by SHFRGRP** [comment]
Already changed my issue!

**Comment by tensorflowbutler** [comment]
The original poster has replied to this issue after the stat:awaiting response label was applied.

**Comment by drpngx** [comment]
Unfortunately that's not a configuration that we support.

Are you trying to run the tensorflow python environment or just the runtime?

**Comment by talregev** [comment]
I have the same Issue.
Are you support official in tensorflow-aarch64 package?
https://pypi.org/project/tensorflow-aarch64/

I try just 
'import tensorflow'   
no module named tensorflow".

I am using oneplus 3t android device.
android 8.1.0
gpu not relevant. I want to use cpu version.

**Comment by mohantym** [comment]
We see that you are using old version of Tensorflow which is officially considered as end of life, We recommend that you upgrade to 2.5  version or later and let us know if the issue still persists in newer versions .Please open a new issue in case you face any errors, we will get you the right help .Thanks!

**Comment by talregev** [comment]
I currently not using it. If there will be an issue, I will open a new one.

**Comment by mohantym** [comment]
Hi @SHFRGRP ! @talregev !
Could you check the [tensorflow-aarch64](https://pypi.org/project/tensorflow-aarch64/) from pypi and let us know .
Thank you!
